src/openfl/text/TextField.hx

```diff
@@ -51,84 +53,84 @@ import js.html.DivElement;
 #end
 
 /**
-	The TextField class is used to create display objects for text display and
```

Comment:
Looks like there were some changes to the doc comment style you'd have to merge manually.